src/core/annotation.js

```diff
@@ -16,60 +16,12 @@
            stringToBytes, Promise, isArray, ObjectLoader, OperatorList,
            isValidUrl, OPS, createPromiseCapability, AnnotationType,
            stringToUTF8String, AnnotationBorderStyleType, ColorSpace,
-           AnnotationFlag, isInt */
+           AnnotationFlag, isInt, InteractiveAnnotation */
```

Comment:
This change can we removed: `InteractiveAnnotation` has been removed in #6178 as it was an unnecessary abstraction. Just inherit from `Annotation` directly.